commands/rm.go

```diff
@@ -34,10 +34,11 @@ func cmdRm(c *cli.Context) {
 		if err := mcn.Remove(host, force); err != nil {
 			log.Errorf("Error removing machine %s: %s", host, err)
 			isError = true
+		} else {
+			log.Infof("Successfully removed %s", host)
```

Comment:
So that success is reported immediately after each host is removed, and so that users know _which_ host(s) got removed successfully.